lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -202,7 +202,7 @@ def test_persistent_grades_not_enabled_on_course(self, default_store):
             self.set_up_course(enable_persistent_grades=False)
             self.assertFalse(PersistentGradesEnabledFlag.feature_enabled(self.course.id))
             with check_mongo_calls(0):
-                with self.assertNumQueries(0):
+                with self.assertNumQueries(1):
```

Comment:
Note that snippet from the doc is talking about kwargs passed to the *decorator*, not the apply_async call.  

I looked into adding a new argument to apply_async, but it got complicated, because 

1. The task instance is created once for multiple applications of the task, so any variables set on the task object need to be cleared when the next task runs
2. The apply_async call happens on one server, and the actual application of the task happens on a different one, so you have to figure out how to get the argument onto the server, which might involve injecting it into the arguments called on the task, and then stripping it out again before actually calling the task.  

The more I looked at this kind of solution, the deeper I had to get into the guts of celery, and celery doesn't really seem architected for this kind of adaptation.

Another, perhaps more tractable alternative would be to perform the resolution update in a separate task that gets called on success of the initial task (using celery task chains).